next.config.js

```diff
@@ -3,7 +3,7 @@ const {
   PHASE_PRODUCTION_BUILD,
 } = require("next/constants");
 
-const can = require("./can.config");
+const can = require("./canopy.config");
```

Comment:
Changes here are going to conflict with PR #24.  Want to just close that and handle everything here?